natlas-server/app/host/migrators.py

```diff
@@ -0,0 +1,15 @@
+def determine_data_version(hostdata):
+	if 'agent_version' in hostdata:
+		# Do (bad) math on version here to determine if we need to fall "up" to 0.6.4
+		version = hostdata['agent_version']
+		verlist = version.split('.')
```

Comment:
https://pypi.org/project/version-parser/ Custom parsers are doomed to be buggy.

Reply:
Using https://pypi.org/project/semver/ instead, but agreed that I shouldn't have been trying to do my own version parsing. Especially when we're using a common versioning format that already has parsers available.